websites/W/WoolyPooly/dist/metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+  "author": {
+    "name": "The Gamerzs",
+    "id": "730142125181894657"
+  },
+  "service": "WoolyPooly",
+  "description": {
+    "en": "WoolyPooly is one of the most profitable pools with our focus on minimal commission fees for PPLNS & SOLO reward systems."
+  },
+  "url":
+    "woolypooly.com",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/X2sf9rc.png",
+  "thumbnail": "https://i.imgur.com/RVtEVMf.png",
+  "color": "#3d89f5",
+  "tags": ["other"],
```

Comment:
pretty sure you can come up with some better tags...

Reply:
I thought the tags were only ones on the store

Reply:
nvm, I'm thinking of category